### PR TITLE
WebUI: Add modern login page design (0207-WebUI-ModernLoginPage)

### PR DESCRIPTION
**Pull request by amroeser**
- Split-screen layout: form panel left, hero image right
- User selection as pill buttons with initial avatar
- Glass-morphism inputs with focus ring
- Password show/hide toggle (inline SVG, no external dependencies)
- Modern primary login button with hover/active states
- Fully responsive (mobile: hero panel hidden, form full-width)
- All JS functions, IDs and form field names preserved 1:1
- CSS and JS extracted to separate files (login.css, login.js)

## Summary by CodeRabbit

* **New Features**
  * Modernized login page with panel-based layout and refreshed visual styles
  * Clickable user-selection pills for quick account selection
  * Password visibility toggle and Enter-to-submit support for easier login
  * Inline error display for clearer feedback
  * Improved help interaction and accessible labels/placeholders
  * Responsive layout; hero/testimonial content hidden on narrow viewports

### CODE REVIEW

**Comment by jens-maus** [comment]
First of all, thank you @amroeser for this PR. However, before considering merging your proposed PR, please address the following:

1. Provide a screenshot or even a screencast demonstrating your proposed change to the login page/dialog
2. Please address all coderabbitai suggestions including the nitpicking part.

After you have provided these additional changes/information I will then happily start to review your PR.

**Comment by amroeser** [comment]
## Screenshots – Modern Login Page

**Desktop (Split-Screen, 1400×860):**

Desktop Login

**Mobile (390×844, Hero ausgeblendet):**

Mobile Login

---

All coderabbitai review comments addressed in previous commits:
- Local hero image (no external Unsplash dependency)
- `vh`/`vw` fallbacks for `100dvh`/`100dvw`
- Media queries reordered desktop-first (1200px → 768px)
- `aria-label` uses i18n token `${lblShowPassword}`
- Help text uses i18n token `${lblHelp}`
- XSS fix: `data-fullname`/`data-username` attributes + JS listener
- `#content` div added for `Ajax.Updater` target
- `<div id=btnLogin>` → `<button type=button>` for keyboard/a11y
- Testimonial texts use i18n tokens `${lblTestimonialHomeMatic}` etc.

File: buildroot-external/patches/occu/0207-WebUI-ModernLoginPage/occu/WebUI/www/rega/login.css [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+/* ── Layout ─────────────────────────────────────────────── */
+#maintable {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+  width: 100vw;
+  height: 100dvh;
+  width: 100dvw;
+  overflow: hidden;
+}
```
[the remark this answers is not in the document]
Comment (amroeser) [reply buildroot-external/patches/occu/0207-WebUI-ModernLoginPage/occu/WebUI/www/rega/login.css:18]: I'll check it out tomorrow…

**Comment by MichaelN0815** [comment]
Just my 2 ct: picture and coloring doesn't match to the WebUI.

**Review by jens-maus — CHANGES_REQUESTED** [review]
Thanks again @amroeser for this PR and the interesting screenshots. However, I do have the following requests/concerns regarding this PR, which we should start to discuss:

1. Please address all the rest of the coderabbitai comments and nitpicks as they are usually quite helpful in identifying issues in long term.
2. Please remove the `screenshot-desktop.png` and `screenshot-mobile.png` files from this PR, but simply copy&paste them to the initial PR description so that anyone can immediately find them and that they will not be part of your repository submission
3. Regarding you suggested redesign, I would be interested to know where you grabbed the `login-hero.jpg` file from since we have to take special care that this image will be from an authentic source with an appropriate license that is compatible to the general Apache-2.0 we use in this project.
4. Your current design is not picking up the current OpenCCU logo. So I would suggest to instead of just showing a checkmark with the "OpenCCU" text, to better display the general OpenCCU logo there at least.
5. Besides the above points, I also do have the general feeling (similar to @MichaelN0815) that your proposed redesign does not fit the general design of the rest of the OpenCCU WebUI. So instead of just patching/modernizing the login page, I would like to elaborate on your plan on how to also update the rest of the core WebUI? IMHO it would be better to first work on a plan/idea on how to modernize the WebUI (e.g. we also do have the bootstrap css classes available in the current web framework) before we actually can merge this PR to modernize the login page. Because otherwise users will certainly be irritated if they see the modernized login page, but then after login will see the old-fashioned webui instead. So perhaps we should start discussing on your ideas (perhaps in the discussion fora) in case you have the time, motivation and technical knowledge on how to modernize the general WebUI.
